src/main/scala/com/microsoft/partnercatalyst/fortis/spark/pipeline/BingPipeline.scala

```diff
@@ -15,9 +17,11 @@ object BingPipeline extends Pipeline {
 
   private def convertToSchema(stream: DStream[BingPost], transformContext: TransformContext): DStream[AnalyzedItem] = {
     stream.map(post => AnalyzedItem(
+      createdAtEpoch = now.getEpochSecond,
```

Comment:
Question: Do these incoming posts have a date of their own? If so, do we prefer to use our own capture time as opposed to the timestamp in the post?